classic-commerce.php

```diff
@@ -21,29 +21,48 @@
 }
 
 /**
- * Returns error when WooCommerce is detected among the files on the server.
+ * Shows an error message when WooCommerce is detected as currently active.
+ *
+ * WooCommerce and Classic Commerce cannot both be active at once.
  *
  * @return void
  */
 function cc_wc_already_active_notice() {
-	echo '<div class="error notice is_dismissible"><p>';
+	echo '<div class="notice error is-dismissible">';
+	echo '<p style="font-weight: bold">';
 	echo esc_html__( 'You must deactivate WooCommerce before activating Classic Commerce.', 'classic-commerce' );
-	echo '</p></div>';
+	echo '</p>';
+	echo '<p>';
+	echo esc_html__( 'Classic Commerce has been deactivated.', 'classic-commerce' );
```

Comment:
As a user I am trying to activate. Wouldn't saying, "Classic Commerce is still inactive. You must deactivate WooCommerce before activating Classic Commerce." give more meaning?

Reply:
Yes this makes sense. I think it should be like this then:

>**You must deactivate WooCommerce before activating Classic Commerce.**
>
>Classic Commerce has not been activated.